GTG/core/timer.py

```diff
@@ -70,15 +72,64 @@ def time_changed(self):
         self.timeout_source = GObject.timeout_add_seconds(secs_to_refresh,
                                                           self.emit_refresh)
 
-    def set_configuration(self, refresh_hour, refresh_min):
+    def set_configuration(self, time):
+        self.config.set('hour', time.hour)
+        self.config.set('min', time.minute)
+
+    def get_configuration(self):
         try:
-            datetime.time(int(refresh_hour), int(refresh_min))
-        except (ValueError, TypeError):
-            raise ValueError("{}:{} is not a valid time".format(refresh_hour,
-                                                                refresh_min))
+            return datetime.time(int(self.config.get('hour')),
+                                 int(self.config.get('min')))
+        except(ValueError):
+            Log.error("Invalid time values. Try using HH:MM or am/pm format")
```

Comment:
Can you log the values you got out of the configuration? It would help with debugging issues.